flutter_local_notifications/lib/src/platform_flutter_local_notifications.dart

```diff
@@ -255,6 +256,19 @@ class AndroidFlutterLocalNotificationsPlugin
   Future<void> deleteNotificationChannel(String channelId) =>
       _channel.invokeMethod('deleteNotificationChannel', channelId);
 
+  Future<List<ActiveNotification>> getActiveNotifications() async {
+    final List<Map<dynamic, dynamic>> activeNotifications =
+        await _channel.invokeListMethod('getActiveNotifications');
+    return activeNotifications
+        .map((activeNotification) => ActiveNotification(
```

Comment:
Same comment as above on variable name. Also running `flutter analyze` shows an error where the type hasn't been specified

Reply:
I've added `// ignore: always_specify_types` before that line as adding the type conflicts with another rule